### PR TITLE
Fix tables column width

### PR DESCRIPTION
**Pull request by katyafervent**
* Add table column width
* Reduce status column width, since it is only one icon

### CODE REVIEW

File: kqueen_ui/blueprints/ui/templates/ui/organization_manage.html [hunk 1]
```diff
@@ -14,20 +14,20 @@ <h3>Members</h3>
...
   <table class="table table-hover">
     <thead>
-      <th>Name</th>
-      <th>E-mail</th>
-      <th>Role</th>
-      <th>Created</th>
+      <th class="col-md-5">Name</th>
+      <th class="col-md-5">E-mail</th>
+      <th class="col-md-3">Role</th>
+      <th class="col-md-4">Created</th>
```
Comment (ekhomyakova) [inline kqueen_ui/blueprints/ui/templates/ui/organization_manage.html]: AFAIK sum of all `col-md-*` should equal to 12. Here is 17, and last 2 columns don't even have any...

File: kqueen_ui/asset/dynamic/scss/main.scss [hunk 2]
```diff
@@ -479,7 +480,7 @@ table {
   .table {
     @include table-col-x;
     .status_column {
-      width: 160px;
+      //width: 80px;
```
Comment (ekhomyakova) [inline kqueen_ui/asset/dynamic/scss/main.scss]: Are you going to remove this?

**Comment by ekhomyakova** [comment]
Looks fine, if the tables are still ok after that

**Comment by naumvd95** [comment]
pls attach screenshot

**Comment by katyafervent** [comment]
@naumvd95 It's not just a one screenshot, except bug fixing (with PM\AM get down from the main text) the main improvement is on the cluster details page, You can compare before\after by loading  update-table-layout branch (and running  gulp build)

**Pull request merged**